platform/persistence/platform-persistence-commands/src/main/java/org/codice/ddf/persistence/commands/StoreDeleteCommand.java

```diff
@@ -17,15 +17,22 @@
 import java.util.List;
 import java.util.Map;
 
-import org.apache.felix.gogo.commands.Command;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.apache.karaf.shell.api.console.Session;
 import org.codice.ddf.persistence.PersistenceException;
 
 import jline.console.ConsoleReader;
 
+@Service
 @Command(scope = "store", name = "delete",
         description = "Deletes entries from the persistence store.")
 public class StoreDeleteCommand extends AbstractStoreCommand {
 
+    @Reference
+    Session session;
```

Comment:
StoreDeleteCommand was the only file that needed to use `session`, so I thought it would be best to add it there. The code that I used as reference (in catalog-core-commands) did the same thing.